examples/android/CHIPTool/app/src/main/java/com/google/chip/chiptool/setuppayloadscanner/BarcodeFragment.kt

```diff
@@ -178,24 +178,24 @@ class BarcodeFragment : Fragment() {
     }
   }
 
-  private fun handleInputQrCode(qrCode: String) {
+  private fun handleInputCode(code: String) {
     lateinit var payload: OnboardingPayload
```

Comment:
Does payload need to be lateinit anymore? We could inline it on line 184

Reply:
@andy31415  Thank you for a good idea. I removed lateinit because it was not required. 
Additionally, i modified the class to match the kotlin format. Please let me know if you need anything else.